engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -130,4 +129,29 @@ public void initCounters() {
         } while (rbacUsers.getData().size() == rbacElementsPerPage);
         return users;
     }
+
+    private List<User> transformToUser(List<ITUserResponse> itUserResponses, boolean adminsOnly) {
+        List<User> users = new LinkedList<>();
+        for (ITUserResponse itUserResponse : itUserResponses) {
```

Comment:
Will all fields used in the following code always be populated? There are several potential NPE otherwise.